README.md

```diff
@@ -151,7 +154,11 @@ Steerage also uses [confidence](https://github.com/hapijs/confidence) to give yo
 }
 ```
 
-Here you can see the `$filter` and `$default` fields.  These fields allow for filtering on a resolver like `env.NODE_ENV`. The `$filter` field evaluates the environment variable `NODE_ENV`. Then it will look to the following fields for a match in the keys for that value, otherwise the `$default` value is used. So the configuration values and options for plugins will change based on the environment variable `NODE_ENV`. You could also determine whether plugin should be registered at all. See the code below for an example based on this `manifest.json` file.
+in this example, the `$filter` and `$default` fields allow for filtering based on a resolver like `env.NODE_ENV`. 
+
+The `$filter` field evaluates the environment variable `NODE_ENV`. Then, it will look to the following fields for a match in the keys for that value. Otherwise, the `$default` value is used. So the configuration values and options for plugins will change based on the environment variable `NODE_ENV`.
+
+You could also determine whether a plugin should be registered at all. See the code below for an example based on this `manifest.json` file.
```

Comment:
>You could also determine whether a plugin should be registered at all. See the code below for an example based on this `manifest.json` file.

I feel like this first sentence implies there's an example of this. Maybe add one? Non-blocking.

Reply:
I added an example 👍